From f119eac253f2191d307f556264eb9c837411177c Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Tue, 10 Jul 2018 17:39:36 +0300
Subject: [PATCH] Add uboot-test-hooks support fixup

update SPL image size due to additional images of SPL (64KBx2)
---

diff --git a/board/habanalabs/common/t32/prog_spi.cmm b/board/habanalabs/common/t32/prog_spi.cmm
index ac9a5dd..bb48026 100644
--- a/board/habanalabs/common/t32/prog_spi.cmm
+++ b/board/habanalabs/common/t32/prog_spi.cmm
@@ -58,10 +58,10 @@
 
   ;FLASHFILE.DUMP 0x0
 
-  FLASHFILE.unlock 0x0--0x27FFFF
-  FLASHFILE.erase 0x0--0x27FFFF
-  FLASHFILE.load /lib/firmware/habanalabs/dali/dali-spi-flash.bin 0x0
-  FLASHFILE.load /lib/firmware/habanalabs/dali/dali-spi-flash.bin 0x0  /ComPare
+  FLASHFILE.unlock 0x0--0x29FFFF
+  FLASHFILE.erase 0x0--0x29FFFF
+  FLASHFILE.load /lib/firmware/habanalabs/dali/dali-spi-flash.bin 0x0++0x29FFFF
+  FLASHFILE.load /lib/firmware/habanalabs/dali/dali-spi-flash.bin 0x0++0x29FFFF  /ComPare
 
 ENDDO
 
